cilium/cmd/policy.go

```diff
@@ -114,14 +115,19 @@ func ignoredFile(name string) bool {
 func loadPolicyFile(path string) (api.Rules, error) {
 	var content []byte
 	var err error
+	var r io.Reader
 	logrus.WithField(logfields.Path, path).Debug("Loading file")
 
 	if path == "-" {
-		content, err = io.ReadAll(bufio.NewReader(os.Stdin))
+		r = bufio.NewReader(os.Stdin)
 	} else {
-		content, err = os.ReadFile(path)
+		f, err := os.Open(path)
```

Comment:
nit, could just be 
```suggestion
		r, err = os.Open(path)
```
to drop the `r = f` below (since we have a `var err error` anyway)